package.json

```diff
@@ -1,11 +1,13 @@
 {
   "name": "k2-extension",
-  "version": "1.3.65",
+  "version": "1.3.66",
   "description": "A Chrome Extension for Kernel Schedule",
   "private": true,
   "scripts": {
     "preinstall": "./tools/checkRuntimeVersions.sh",
     "build": "webpack --progress --config webpack.prod.js",
+    "build:chrome": "webpack --progress --config webpack.prod.js --env platform=chrome",
```

Comment:
Is the chrome build necessary? If so, let's be sure to update the README instructions for that too (and maybe remove the `build` command completely).

Reply:
it's not necessary, but my intention was to leave the default build command as it was, maintaining Chrome compatibility, add a new one for Firefox, and have `build:chrome` as an alias for build just for clarity

Reply:
Made a change to make it clearer that `build` is just an alias for `build:chrome`